controller/ca/controller.go

```diff
@@ -20,24 +23,47 @@ import (
 type CertificateController struct {
 	namespace   string
 	k8sAPI      *k8s.API
+	ca *CA
+	trustAnchorsPEM string
 	syncHandler func(key string) error
+
+	// The queue is keyed on a string. If the string doesn't contain any dots
+	// then it is a namespace name and the task is to create the CA bundle
+	// configmap in that namespace. Otherwise the string must be of the form
+	// "$podOwner.$podKind.$podNamespace" and the task is to create the secret
+	// for that pod owner.
 	queue       workqueue.RateLimitingInterface
 }
 
-func NewCertificateController(conduitNamespace string, k8sAPI *k8s.API) *CertificateController {
+func NewCertificateController(conduitNamespace string, k8sAPI *k8s.API) (*CertificateController, error) {
+	ca, err := NewCA()
+	if err != nil {
+		return nil, err
+	}
+
 	c := &CertificateController{
 		namespace: conduitNamespace,
 		k8sAPI:    k8sAPI,
+		ca: ca,
+		trustAnchorsPEM: ca.TrustAnchorPEM(),
 		queue: workqueue.NewNamedRateLimitingQueue(
 			workqueue.DefaultControllerRateLimiter(), "certificates"),
 	}
 
-	k8sAPI.Pod().Informer().AddEventHandler(
-		cache.ResourceEventHandlerFuncs{
-			AddFunc:    c.handlePodAdd,
-			UpdateFunc: c.handlePodUpdate,
-		},
-	)
+	// TODO: Handle deletions.
+	handlePodOwnerAddUpdate := cache.ResourceEventHandlerFuncs{
```

Comment:
> Is this local var necessary?

I factored it out because I figured we'd reuse the same `handlePodOwnerAddUpdate` for every informer (ReplicatSets, Deployments, etc.)